src/SFA.DAS.FindEmploymentSchemes.Contentful/Content/GeneratedContent.cs

```diff
@@ -0,0 +1,312 @@
+                null,
+                null,
+                null,
+                null,
+                null,
+                "How to offer",
+                null
+                ),
+        };
+       private Filter? _motivationsFilter;
+        public Filter MotivationsFilter => _motivationsFilter ??= new Filter("motivations", "I want to", new FilterAspect[]
+        {
+            new FilterAspect("motivations--full-time-role", "help train someone up into a full-time role"),
+            new FilterAspect("motivations--unpaid-placement", "offer someone an unpaid work placement opportunity"),
+            new FilterAspect("motivations--diversity-or-responsibility", "diversify our workforce or for corporate and social responsibility"),
+        });
+       private Filter? _payFilter;
+        public Filter PayFilter => _payFilter ??= new Filter("pay", "I can offer", new FilterAspect[]
+        {
+            new FilterAspect("pay--minimum-wage", "at least national minimum wage"),
+            new FilterAspect("pay--unpaid", "unpaid placements"),
+        });
+       private Filter? _schemeLengthFilter;
+        public Filter SchemeLengthFilter => _schemeLengthFilter ??= new Filter("schemeLength", "Length of scheme?", new FilterAspect[]
+        {
+            new FilterAspect("scheme-length--up-to-4-months", "Up to 4 months"),
+            new FilterAspect("scheme-length--4-months-to-12-months", "4 months to 12 months"),
+            new FilterAspect("scheme-length--a-year-or-more", "A year or more"),
+        });
+        private IEnumerable<Page>? _pages;
+        public IEnumerable<Page> Pages => _pages ??= new Page[]
+        {
+            new Page("Cookies",
```

Comment:
The generated content doesn't show the marketing and analytics page content - does this matter now contenful is integrated?

Reply:
Yes, the generated content allows the web site to come up, even if the Contentful API is down, so we need to keep the backup generated content up to date.

The generated content is now up to date.